### DISCUSSION TITLE
Unable to display 2 ngx-extended-pdf-viewer in a single page

### DISCUSSION THREAD

**Discussion by shakyamoksha** [stephanrauh/ngx-extended-pdf-viewer#1674] [General]
Hi, thank you for this module, been using it for 3 years now.

However, I am not able to display 2 ngx-extended-pdf-viewer in a single page, I have even followed your showcase sample of using iFrames, but the implementation doesn't work, meaning, the iFrames is always empty.

Would it be possible for you to publish a working StackBlitz repo with this special implementation?

thank you in advance

**Comment by stephanrauh** [comment]
I'm afraid I can't. Last time I check, Stackblitz was incompatible to ngx-extended-pdf-viewer. It can't load the JavaScript files.

However, you can check out the showcase. It's always possible my instructions are wrong, but the showcase is guaranteed to run. Usually comparing my working demo to your sourcecode helps you to spot the difference quickly. Here you go: 

- git clone https://github.com/stephanrauh/extended-pdf-viewer-showcase.git
- The demo is here: https://github.com/stephanrauh/extended-pdf-viewer-showcase/tree/main/src/app/extended-pdf-viewer/side-by-side

**Reply by stephanrauh** [reply]
BTW, three years... so you've seen the 1.x versions. Awesome!
